homeassistant/components/poolsense/__init__.py

```diff
@@ -0,0 +1,76 @@
+"""The PoolSense integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+import poolsense
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client, update_coordinator
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the PoolSense component."""
+    # Make sure coordinator is initialized.
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up PoolSense from a config entry."""
+
+    await get_coordinator(hass, entry)
+
+    if not entry.unique_id:
```

Comment:
can there be more than one poolsesne device on account?

Reply:
Yes. The username and password return the full list of devices associated with that user account. So the email should be unique for every set of PoolSense devices.

Reply:
Currently this integration will only return 1 or the first one you have (in a multiple installation setup). Most home users will only have one, a corporate client may have multiple. I will make sure this is mentioned in the documentation. I will update the integration to allow for multiple as I progress.